lib/cylc/cfgspec/globalcfg.py

```diff
@@ -212,47 +249,75 @@
     },
 
     'authentication': {
-        # Allow owners to grant public shutdown rights at the most, not full control.
+        # Allow owners to grant public shutdown rights at the most, not full
+        # control.
         'public': vdr(
             vtype='string',
-            options=PRIVILEGE_LEVELS[:PRIVILEGE_LEVELS.index('shutdown')+1],
+            options=PRIVILEGE_LEVELS[:PRIVILEGE_LEVELS.index('shutdown') + 1],
             default="state-totals")
     },
 }
 
-def upg( cfg, descr ):
-    add_bin_dir = converter( lambda x: x + '/bin', "Added + '/bin' to path" )
-    use_ssh = converter( lambda x: "ssh", "set to 'ssh'" )
-    u = upgrader(cfg, descr )
-    u.deprecate( '5.1.1', ['editors','in-terminal'], ['editors','terminal'] )
-    u.deprecate( '5.1.1', ['task hosts'], ['hosts'] )
-    u.deprecate( '5.1.1', ['hosts','local'], ['hosts','localhost'] )
-    u.deprecate( '5.1.1', ['hosts','__MANY__', 'workspace directory'], ['hosts','__MANY__', 'workdirectory'] )
-    u.deprecate( '5.1.1', ['hosts','__MANY__', 'cylc directory'], ['hosts','__MANY__', 'cylc bin directory'], add_bin_dir )
-    u.obsolete(  '5.2.0', ['hosts','__MANY__', 'cylc bin directory'], ['hosts','__MANY__', 'cylc bin directory'] )
-    u.deprecate( '5.2.0', ['hosts','__MANY__', 'use ssh messaging'], ['hosts','__MANY__', 'task communication method'], use_ssh )
-    u.deprecate( '6.1.2', ['task messaging', 'connection timeout in seconds'], ['task messaging', 'connection timeout'] )
-    u.deprecate( '6.1.2', ['task messaging', 'retry interval in seconds'], ['task messaging', 'retry interval'] )
-    u.deprecate('6.4.0',
+
+def upg(cfg, descr):
+    add_bin_dir = converter(lambda x: x + '/bin', "Added + '/bin' to path")
+    use_ssh = converter(lambda x: "ssh", "set to 'ssh'")
+    u = upgrader(cfg, descr)
+    u.deprecate('5.1.1', ['editors', 'in-terminal'], ['editors', 'terminal'])
```

Comment:
It'd be nicer if these were stylistically consistent with the multi-line versions below.